backend/api/serializers.py

```diff
@@ -0,0 +1,278 @@
+            .get_object_for_this_type(id=validated_data.pop('object_id'))
+        comment = Comment.objects.create(owner=owner, content_object=content_object, **validated_data)
+        return comment
+
+
+class CommentDetailsSerializer(serializers.ModelSerializer):
+    content_type = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Comment
+        fields = ('id', 'owner', 'content_type', 'object_id', 'content', 'created', 'updated')
+        read_only_fields = ('owner', 'content_type', 'object_id', 'created', 'updated')
+
+    def get_content_type(self, obj):
+        return obj.content_type.model
+
+
+class FollowCreateSerializer(serializers.ModelSerializer):
+    content_type = serializers.CharField()
+
+    class Meta:
+        model = FollowStatus
+        fields = ('id', 'content_type', 'object_id')
+
+    def create(self, validated_data):
+        owner = self.context.get("request").user
+        content_object = ContentType.objects.get(model=validated_data.pop('content_type')) \
+            .get_object_for_this_type(id=validated_data.pop('object_id'))
+        follow_status = FollowStatus.objects.create(owner=owner, content_object=content_object, **validated_data)
+        return follow_status
+
+
+class FollowDetailsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = FollowStatus
+        fields = ('id', 'owner')
+
+
+class MediaDependentCreateSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Media
+        fields = ('id', 'url')
+
+
+class MediaCreateSerializer(serializers.ModelSerializer):
+    content_type = serializers.CharField()
+
+    class Meta:
+        model = Media
+        fields = ('id', 'url', 'content_type', 'object_id')
+
+    def create(self, validated_data):
+        owner = self.context.get("request").user
+        content_object = ContentType.objects.get(model=validated_data.pop('content_type')) \
+            .get_object_for_this_type(id=validated_data.pop('object_id'))
+        media = Media.objects.create(owner=owner, content_object=content_object, **validated_data)
+        return media
+
+
+class MediaDetailsSerializer(serializers.ModelSerializer):
+    content_type = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Media
+        fields = ('id', 'owner', 'content_type', 'object_id', 'url', 'created', 'updated')
+
+    def get_content_type(self, obj):
+        return obj.content_type.model
+
+
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        fields = ('id', 'name')
+
+
+class VoteCreateSerializer(serializers.ModelSerializer):
+    content_type = serializers.CharField()
+
+    class Meta:
+        model = VoteStatus
+        fields = ('id', 'content_type', 'object_id', 'vote')
+
+    def create(self, validated_data):
+        owner = self.context.get("request").user
+        content_type_object = ContentType.objects.get(model=validated_data['content_type'])
```

Comment:
`.get(item)` is better.

Reply:
in the following lines as well.

Reply:
Actually, it doesn't matter in out situation because it that field is missing, validators catch that and code don't even react this point.